composer.json

```diff
@@ -89,7 +89,7 @@
     },
     "conflict": {
         "symfony/dependency-injection": "5.3.7",
-        "symfony/security-http": "v5.3.0 v5.3.1",
+        "symfony/security-http": "v5.3.0 || v5.3.1",
```

Comment:
Symfony 5.3 is EOL. All Symfony 5.3 requirement can be replaced by 5.4.

See : https://github.com/bolt/core/pull/3535